contents/handbook/company/communication.md

```diff
@@ -31,7 +31,7 @@ We make things public by default because [transparency is part of Communication,
 
 - _Public_ - most things, including our product, roadmap, handbook and strategy. 
 - _Shared internally_ - almost everything else, such as financial performance, security, fundraising and recruitment.
-- _Private internally_ - personal team information, ie. compensation, disciplinary issues.
+- _Private internally_ - personal team information, i.e. compensation, disciplinary issues.
```

Comment:
The term [i.e.](https://www.merriam-webster.com/dictionary/i.e.) should include a period between the letters "i" and "e".